back/sssscs/src/main/java/com/ib/user/User.java

```diff
@@ -0,0 +1,25 @@
+package com.ib.user;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
```

Comment:
Don't use [at]Data for Entitities. Read: https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls

Reply:
Fixed in [526576c](https://github.com/magley/sssscs/pull/1/commits/526576cf20344a23c92be7c7ecad2359058da605). I added `@ToString` because it's convenient and if we run into reference issues later on, we can explicitly ignore those fields.